### PR TITLE
Add org ids stats tables, including to publisher page 

### PR DESCRIPTION
**Pull request by Bjwebb**

### CODE REVIEW

**Comment by Bjwebb** [comment]
Not ready for merge yet.

I've also merged this onto dev, so that I can see what the build looks like.

File: make_html.py [hunk 1]
```diff
@@ -294,6 +295,15 @@ def element(slug):
                            page='elements')
 
 
+@app.route('/org_type/<slug>.html')
+def org_type(slug):
+    assert slug in slugs['org_type']['by_slug']
+    return render_template('org_type.html',
+                           slug=slug,
+                           url=lambda x: '../' + x,
```
Comment (andylolz) [inline make_html.py:303]: I think this lambda is unused currently, so this line could be removed.

**Comment by andylolz** [comment]
Sorry – I realise you’re still working on this, but I’ve added a couple of v minor suggestions.

The other thing I wondered about is blanks instead of zeros. I think elsewhere, tables do include zeros, so I’m in favour of doing that here too.

**Comment by Bjwebb** [comment]
That all makes sense, so I've made the suggested changes.

I've also changed the columns quite a bit, and grouped them, it would be useful to know what you think.
Next I'll apply these changes to the other tables.

Am I okay to replace dev with this branch, so that I can check builds are working okay?

**Comment by Bjwebb** [comment]
CI is failing because coveralls is down.

**Comment by stevieflow** [comment]
> I've also changed the columns quite a bit, and grouped them

I took at look at this in dev and think it's really interesting.  I suggest the order / grouping of the columns is not a blocker though - if we want, we can revise that later, once this is in production and being populated.  Hence, I would not slow this PR down in terms of that factor @andylolz @markbrough (not that you would !)

**Comment by andylolz** [comment]
> Am I okay to replace dev with this branch, so that I can check builds are working okay?

^^ V pleased you did this – it makes it much easier to take a look at it!

I’m a big fan of this grouping – I think it’s a big improvement. In particular, the meaning of the final two columns (valid refs and % valid) is now much clearer to me.

Aesthetically I don’t really like the “self refs” stacked on top of another “self refs” – would you consider vertically merging those cells?

Another aesthetic thing, but would you consider running the percentage column through the [`round_nicely` filter](https://github.com/codeforIATI/analytics/blob/4d6e620772fe35ec2001d67dec2718e5d653386d/make_html.py#L73-L78)?

Both of those are v minor things, and neither is a deal breaker nor should be a blocker to merging!

**Comment by Bjwebb** [comment]
I've made your suggested updates, and applied the same columns to the other org id tables, including on the publisher page. This is now building for the dev site.

Screenshot from 2022-12-09 11-04-50

Screenshot from 2022-12-09 11-05-46

Publisher page:

Screenshot from 2022-12-09 11-05-00

**Review by andylolz — APPROVED** [review]
Nice – this looks good to me!

**Comment by Bjwebb** [comment]
It's live! https://analytics.codeforiati.org/org_ids.html

**Pull request merged**